### PR TITLE
User story 6 upgrading

### PR DESCRIPTION
**Pull request by jdai54**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
```
Comment (aspencer8111) [inline config/routes.rb:1]: This. This right here is a beautiful routes file.

**Review by aspencer8111 — APPROVED** [review]
Looks good. Bonus challenge: When you get time, write some Rspec controller and model specs for all of this funness!

File: config/routes.rb (hunk 1 above)
Comment (jdai54) [reply config/routes.rb:1]: 👍

**Review by aspencer8111 — COMMENTED** [review]
👍

**Pull request merged**